### ISSUE TITLE
Make adaptation for webstorm 

### ISSUE THREAD

**Issue by OrenMag** [mattpocock/ts-error-translator#47]
Would be highly appreciated :)

**Comment by mattpocock** [comment]
I agree! I won't prioritise this myself, but I'm very willing to review a PR for someone to contribute this.

**Comment by ahmadshiddiq** [comment]
this stuff really cool, even jetbrains didn't have it.
can't wait until it supported on webstorm 👍

**Comment by th3fallen** [comment]
i would kill for this on jetbrains!

**Comment by alvarogfn** [comment]
up!
